### DISCUSSION TITLE
Is there an alternative way for Channel.getManager().sync() in JDA 5.0.0?

### DISCUSSION THREAD

**Discussion by deso85** [discord-jda/JDA#2147] [Questions and Help] [answered]
Hello everyone,

I tried to migrate from JDA 4.4.0 to 5.0.0 and managed to get everything to work but a function which is used to sync channels permissions with it's parents category.

In 4.4.0 I was able to use this:
_TextChannel_.getManager().**sync()**.queue();

In 5.0.0 I can't find a function to sync the permissions with another Channel or Category. Is there someone who can help me with that?

**Comment by Tais993** [comment, marked as answer]
```java
textChannel.getManager().sync().queue();
```
Is the code

Are you sure that you're running it on a TextChannel? See the [sync method on TextChannelManager](https://ci.dv8tion.net/job/JDA5/javadoc/net/dv8tion/jda/api/managers/channel/attribute/ICategorizableChannelManager.html#sync())

**Reply by deso85** [reply]
Thanks. I don't know what happened but I guess something went wrong after I updated the maven dependencies. I switched back (4.4.0) and forth (5.0.0) and now it works as before.
